src/DynamoCore/DynamoCore.csproj

```diff
@@ -75,9 +75,9 @@ limitations under the License.
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\extern\CSharpAnalytics\CSharpAnalytics.WinForms.dll</HintPath>
     </Reference>
-    <Reference Include="Greg, Version=1.0.5953.24366, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Greg.1.0.5953.24366\lib\net45\Greg.dll</HintPath>
-      <Private>False</Private>
+    <Reference Include="Greg, Version=1.0.5959.26179, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Greg.1.0.5959.26179\lib\net45\Greg.dll</HintPath>
+      <Private>True</Private>
```

Comment:
@mjkkirschner @ikeough , didn't we add them and set copylocal = false for purpose before, Or nuget in VS is in our way of this process again?